db/migrate/20181102141329_add_ssl_options_to_endpoints.rb

```diff
@@ -0,0 +1,7 @@
+class AddSslOptionsToEndpoints < ActiveRecord::Migration[5.1]
+  def change
+    add_column :endpoints, :verify_ssl,            :integer
+    add_column :endpoints, :security_protocol,     :string
```

Comment:
@bdunne I added this here but I think scheme, verify_ssl, and certificate_authority cover what we need.  I think security_protocol in MIQ is the equivalent of scheme here, just instead of ssl/non-ssl we have http/https.  Think we need both?

Reply:
I think we should choose `verify_ssl` or `security_protocol`.
Do we need anything other than these:
```ruby
OpenSSL::SSL::VERIFY == :verify_ssl => true,
OpenSSL::SSL::VERIFY_NONE == :verify_ssl => false,
OpenSSL::SSL::VERIFY_PEER == :verify_ssl => true with :certificate_authority,
```

Reply:
Nope that should be enough.  I agree I'll stick with just verify_ssl and certificate authority

Reply:
Done